backend/backend/appsettings.json

```diff
@@ -10,5 +10,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "EmailConfiguration": {
+    "EmailAddress": "wedontbyte.team@gmail.com",
```

Comment:
Did you actually create the account or is this just placeholder?

Reply:
![image](https://user-images.githubusercontent.com/55400760/113327520-bda86400-9323-11eb-8f42-fda569d76fb8.png)
![image](https://user-images.githubusercontent.com/55400760/113327593-d3b62480-9323-11eb-91c7-36927854d8d6.png)

Reply:
Created the account, tested, works.

Reply:
Moved EmailService to separate folder Services/EmailService/EmailService.cs and IEmailService.cs

Reply:
Okay, one last thing then - I think it'd be a good idea to leave the password field blank here. Yes, our operation isn't serious so if anyone were to steal this account it wouldn't be a big deal, but still, to avoid disruption to the project from some bot that scours Github repositories for emails + passwords or any other thing, I think it'd be good to leave it as an empty string and just post it in our group chat. 

Sadly we don't use Slack so it can't be pinned but when people will actually need to use it (probably rarely) they can ask you (or search) and put it in themselves.

Orrrrr, if we wanna be really "enterprise" we could do it like this: https://dusted.codes/dotenv-in-dotnet

But I'm not sure. What do you think?